sonic-ycabled/ycable/ycable_utilities/y_cable_helper.py

```diff
@@ -3872,7 +3872,7 @@ def task_cli_worker(self):
                     break
 
                 if fvp:
-                    handle_config_hwmode_state_cmd_arg_tbl_notification(fvp, self.cli_table_helper.xcvrd_config_hwmode_state_cmd_sts_tbl,  self.cli_table_helper.xcvrd_config_hwmode_state_rsp_tbl, asic_index, port)
+                    handle_config_hwmode_state_cmd_arg_tbl_notification(fvp, self.cli_table_helper.xcvrd_config_hwmode_state_cmd_sts_tbl,  self.cli_table_helper.xcvrd_config_hwmode_state_rsp_tbl, self.cli_table_helper.hw_mux_cable_tbl, asic_index, port)
```

Comment:
https://github.com/vdahiya12/sonic-platform-daemons/blob/8b6f182b263316fd3c715081c2d865f8b65318c0/sonic-ycabled/ycable/ycable_utilities/y_cable_helper.py#L3097

Is there a syntax error here? How many tables should `handle_config_hwmode_state_cmd_arg_tbl_notification` take as parameters?